src/main/java/com/pgs/booking/controller/UserController.java

```diff
@@ -0,0 +1,43 @@
+package com.pgs.booking.controller;
+
+import com.pgs.booking.model.dto.CreateUserDto;
+import com.pgs.booking.model.dto.UserDto;
+import com.pgs.booking.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/users")
+public class UserController {
+
+    private final UserService userService;
+
+    @PostMapping
+    public UserDto addUser(@Valid @RequestBody CreateUserDto createUserDto) {
+        return userService.addUser(createUserDto);
+    }
+
+    @GetMapping({"/{id}"})
```

Comment:
If we provide just one value we can omit brackets 
```suggestion
    @GetMapping("/{id}")
```